cristin-commons/src/main/java/no/unit/nva/cristin/common/client/CristinQueryApiClient.java

```diff
@@ -1,9 +1,10 @@
 package no.unit.nva.cristin.common.client;
 
+import no.unit.nva.client.GenericApiClient;
 import no.unit.nva.cristin.model.SearchResponse;
 import nva.commons.apigateway.exceptions.ApiGatewayException;
 
-public interface QueryApiClient<T, R> extends GenericApiClient {
+public interface CristinQueryApiClient<T, R> extends GenericApiClient {
```

Comment:
was more tightly coupled to cristin than i thought. gave it a name which reflected that